core/src/miner/mem_pool.rs

```diff
@@ -600,61 +585,13 @@ impl MemPool {
 
     /// Checks the timelock of transactions starting from `start_seq`.
     /// Returns the next seq of the last transaction which can be in the current queue
-    fn check_transactions(
-        &self,
-        public: Public,
-        mut start_seq: u64,
-        current_block_number: PoolingInstant,
-        current_timestamp: u64,
-    ) -> u64 {
+    fn check_transactions(&self, public: Public, start_seq: u64) -> u64 {
```

Comment:
Why does this function still remain?
It seems that this function was introduced because of a timelock.

Reply:
I think this function can be removed, but I couldn't since I didn't fully understand the mempool implementation yet.
It's returning some value derived from `start_seq`, and I'm not sure what it means.